.bazelrc

```diff
@@ -26,7 +26,6 @@ build --incompatible_objc_compile_info_migration
 build --features=-per_object_debug_info
 # Suppress deprecated declaration warnings due to extensive transitive noise from protobuf.
 build --copt -Wno-deprecated-declarations
-build --@envoy//bazel:http3=False
```

Comment:
will this mean we have to exempt quic tests from the release build somehow?
doable but it may make more sense to have this as a lyft-only thing, WDYT?

Reply:
talked offline. Happy to not compile h3 in our private build.